Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.css

```diff
@@ -85,3 +85,27 @@ body:not(.window-inactive, .window-docked-inactive) .content-view.dom-tree .tree
 .content-view.dom-tree .tree-outline.dom li.expanded .status-image.breakpoint.subtree {
     display: none;
 }
+
+.user-preferences-content {
```

Comment:
Can we use a more specific selector for all of these these? At the very least include the popover class?

Reply:
Moved to a new file `OverrideUserPreferencesPopover.css` and made it slightly more specific with `.popover` prefix.